app/src/main/java/org/schabi/newpipe/local/playlist/LocalPlaylistFragment.java

```diff
@@ -381,10 +381,11 @@ public void sharePlaylist() {
         disposables.add(playlistManager.getPlaylistStreams(playlistId)
                 .flatMapSingle(playlist -> Single.just(playlist.stream()
                         .map(PlaylistStreamEntry::getStreamEntity)
-                        .map(StreamEntity::getUrl)
-                        .collect(Collectors.joining("\n"))))
+                        .map(streamEntity -> streamEntity.getTitle() + " " + streamEntity.getUrl())
+                        .collect(Collectors.joining("\n\n"))))
                 .observeOn(AndroidSchedulers.mainThread())
-                .subscribe(urlsText -> ShareUtils.shareText(requireContext(), name, urlsText),
+                .subscribe(urlsText -> ShareUtils.shareText(
+                        requireContext(), name, name + " \n\n" + urlsText),
```

Comment:
I would undo this part of the change. Sharing text on Android already provides the `title` and `content` fields, so you shouldn't add the title at the beginning of the content again.